### DISCUSSION TITLE
A cleaning up thing with config

### DISCUSSION THREAD

**Discussion by editon96** [PurpurMC/Purpur#560] [__Dismissed__] [answered]
Hey I have often the problem that there are folders and some log files on my Server, I clean all that up every Time.
I thought it was cool to can make it automatic.
My Idea ist that a selcetor are made, that clean up cache and other plugin cashes, after defination or files in a freqenzy you choos.

I thought thats a nice thing so I make here a request.
Thanks for reading, have a nice day!

**Comment by granny** [comment, marked as answer]
I don't think this is a thing that should be done by Purpur. You could probably get this done by setting up a crontab or some other application

**Reply by fredster33** [reply]
+1.

**Reply by editon96** [reply]
This is the thing I don´t find any application, but its okay I can do it by myself every Month.
